flocx_market/cmd/app.py

```diff
@@ -0,0 +1,11 @@
+from flocx_market.api import app
+from flocx_market.db.orm import orm
+from flask_migrate import Migrate
+
+application = app.create_app(app_name='flocx-market')
+migrate = Migrate(application, orm)
+
+if __name__ == '__main__':
+
+    orm.init_app(application)
+    application.run(port=8080, debug=True)
```

Comment:
This should use the common service code that initializes things like the config file, the log, etc; I think @DanNiESh or @ljmcgann is working on that.

Reply:
The common service code works. We can have a discuss and put these codes into your application. @fvukelic